content/docs/tasks/telemetry/distributed-tracing/zipkin/index.md

```diff
@@ -35,6 +38,15 @@ To learn how Istio handles tracing, visit this task's [overview](../overview/).
 1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage` one or more times
     to generate trace information.
 
+    The number of requests you need to send in order to see trace data depends on the sampling rate set at install.
```

Comment:
Same comment as previous file.

Note the `{{ <boilerplate> }}` feature available in the site that makes it so you can keep duplicative content in a single place and have the site infra stamp it N times.

Reply:
Done. Used boilerplate.

Reply:
I'm sorry @geeknoid. You know I'm all for boilerplates but we need to be careful when using them. When used, lists can get broken, flows interrupted, etc. 
@douglas-reid I'm sorry about pulling you in different directions but boilerplates are a new feature that we need to figure out. You could use a boilerplate for the warning for example.